indexer/indexer_tests/trie_test.cpp

```diff
@@ -83,74 +84,82 @@ string DebugPrint(KeyValuePair const & p)
 
 struct KeyValuePairBackInserter
 {
-  vector<KeyValuePair> m_v;
   template <class TString>
-  void operator()(TString const & s, trie::FixedSizeValueReader<4>::ValueType const & rawValue)
+  void operator()(TString const & s, uint32_t const & value)
   {
-    uint32_t value;
-    memcpy(&value, &rawValue, 4);
     m_v.push_back(KeyValuePair(s, value));
   }
+
+  vector<KeyValuePair> m_v;
 };
 
-struct MaxValueCalc
+// The SingleValueSerializer and ValueList classes are similar to
+// those in indexer/string_file_values.hpp but that file
+// is not included to avoid coding_tests's dependency from indexer.
```

Comment:
I don’t think it is a good idea. Coding library and it’s tests are at the top of our dependencies hierarchy, immediately after base library.
In abstract case, some new project should be able to use base + coding (and their tests too) without any other dependencies.